packages/core/mesh/network-manager/src/testing/test-builder.ts

```diff
@@ -150,11 +150,11 @@ export class TestPeer {
   }
 
   async goOffline() {
-    await this._networkManager.setNetworkMode(NetworkMode.OFFLINE);
+    await this._networkManager.setState(ConnectionState.OFFLINE);
```

Comment:
remove -- don't provide multiple ways to do the same thing.
also don't use "go" as a prefix instead "set" -- again consistency for users.